docs/README.md

```diff
@@ -0,0 +1,219 @@
+# Death by 1000 needles
```

Comment:
@Arriven is that deployed by accident?
AFAIR we removed this file some time ago